src/pd/client.rs

```diff
@@ -301,9 +301,12 @@ impl PdClient for RpcClient {
         req.set_keys_read(region_stat.read_bytes);
         req.set_approximate_size(region_stat.approximate_size);
 
-        let now = SystemTime::now();
-        let ts = now.duration_since(UNIX_EPOCH).unwrap().as_secs();
-        req.set_timestamp(ts);
+        let ts = SystemTime::now()
```

Comment:
can you extract a function for these? although this is simple, it takes many lines.